packages/api/src/rpc/index.ts

```diff
@@ -1,9 +1,11 @@
-// import cennzx from './cennzx';
-// import genericAsset from './genericAsset';
-//
-// const interfaces: Record<string, object> = {
-//   cennzx,
-//   genericAsset,
-// };
-//
-// export default interfaces;
+import cennzxRpc from './cennzx';
+import genericAssetRpc from './genericAsset';
+import paymentRpc from './payment';
+
+const interfaces = {
+  cennzx: cennzxRpc,
+  genericAsset: genericAssetRpc,
+  payment: paymentRpc,
```

Comment:
nit: can we rely on an `@polkadot` provided definition for `payment`?

Reply:
Yup true, we should be able. I will investigate why queryInfo is then not part of the payment rpc coming from Polkadot.